chainer/link.py

```diff
@@ -172,6 +172,12 @@ def __str__(self):
             cls=self.__class__.__name__, specs=specs,
         )
 
+    def forward(self):
+        """ This should be overwritten by the specific Link class.
```

Comment:
This looks insufficient as a documentation.
* What does this method do?
* What are arguments and return values?

As @toslunar said, it might be an option to leave it a dead link for now and fix it later.

Reply:
Ok. Leaving as dead link for now.